### ISSUE TITLE
[Visual Bug] Steam Icon Missing from Tray

### ISSUE THREAD

**Issue by vemerce** [pop-os/cosmic-epoch#1126]
**Your system information:**

    Steam client version (build number or date): 1726604483
    Distribution (e.g. Ubuntu): Pop!_OS 24.04 Beta
    Opted into Steam client beta?: [Yes/No] No.
    Have you checked for system updates?: [Yes/No] Yes.
    GPU: Nvidia/Intel/AMD: Nvidia 3070 Ti

**Please describe your issue in as much detail as possible:**

When launching Steam, everything opens like normal. However, the top tray icon (top right of screenshot) is blank. Clicking on the blank area opens the Steam menu like normal, but the Steam icon itself is absent.

**Steps for reproducing this issue:**

   1. Open Steam.
   2. Look for Tray icon at top.
   3. It's not there. =P

_![screenshot-2024-10-14-15-54-20](https://github.com/user-attachments/assets/5a7b4939-b306-4da4-8a3d-f696c2019f7d)_

**Comment by hovo1990** [comment]
With newest update of cosmic, all tray icons dissapeared.

**Comment by michaelBelsanti** [comment]
this is fixed on recent versions 
![image](https://github.com/user-attachments/assets/dbd529db-e66a-4b8e-8c42-00c5a4e09fbe)

**Comment by Archpresident** [comment]
It still does it to me with steam flatpak  

![Image](https://github.com/user-attachments/assets/fc1f283f-da6e-4c49-afd2-79a36217be48)

**Comment by Archpresident** [comment]
yes, installed just today the alpha 5 with all the updates, clean home etc..

**Comment by FF-AntiK** [comment]
I also have this issue with the flatpak version of Steam.
I'm using [ublue-os/cosmic](https://github.com/ublue-os/cosmic) and this are the relevant package versions:
`cosmic-desktop-1.0.0~alpha.5.1^20250130-1.fc41.noarch`
`cosmic-panel-1.0.0~alpha.5.1^git20250123.107f597-1.fc41.x86_64`

**Comment by MagicMusicLive** [comment]
I have the same issue with the pCloud desktop client (AppImage) on Alpha 5 with all updates on the date of writing this. The app works, but I have to click an empty space to get a dropdown. Also, and I don't know if this has to do with anything, I have this issue with the default icon theme and custom icon theme (Kora).

Cheers.

**Comment by Brodie-A** [comment]
Still having this issue with the flatpak on 24.04 Alpha 7.

**Comment by vadim-su** [comment]
The issue still persists

**Comment by frsfahd** [comment]
the issue still exists in fedora cosmic with the flatpak steam

**Comment by kode54** [comment]
Issue persists with Steam runtime version from Arch repos, on cosmic-comp-git 1.0.0.alpha.7.r58.g17d6edb-1 - in fact, all my app tray icons are missing

Edit. Never mind, I forgot that comp isn't the panel. I'll try updating my panel.

**Comment by Veraxus** [comment]
I am having this problem on Cosmic Beta 1.1 (Pop!_OS 24.04 LTS) that was released earlier this evening. I tried both Flatpak and bare metal versions of Steam and both are missing the icon. I tried the usual "missing task tray icon" fixes for Steam - the symlink and flatpak permissions fixes - and none of those work.

**Comment by Tichy** [comment]
Same here. Beta and tried both steam flatpak and deb packages. Blank tray icon.

Update: Now the icon is showing up. I don't know what happened to make it appear. I was using Steam from debian package before and after the change.

**Comment by candsastle** [comment]
I still can't see the Steam tray icon. (I'm using the Flatpak version)

**Comment by FriChaFri** [comment]
It is also not showing up for me to this day. I use steam flatpak

**Comment by BalderKj** [comment]
Just installed pop!_OS 24.04 LTS Beta and no steam flatpak tray icon here.

**Comment by HenrikDK2** [comment]
Still an issue with the Steam flatpak

**Comment by SageChaozu** [comment]
I just upgraded to Pop! 24.04 just now and I can confirm that the Steam icon still does not show up. I just upgraded from 22 without a reinstall/upgrade to Steam.

**Comment by Archpresident** [comment]
To solve the problem, I changed the theme of the icons. I went to the gnomelook website and downloaded a few themes (colloid, reversal, kora and tela circle). Colloid and reversal have solved the steam icon in the tray but the arrows disappear in the system settings and maybe even more but I noticed only that, kora is more complete but the icon in the tiling and floating mode panel becomes a square. The ones I use and on which I don't see any shortcomings etc.. are the  tela
